### PR TITLE
Add module website_container_fluid.

### PR DESCRIPTION
**Pull request by yajo**
It lets the web designer choose easily if he wants to use a `<div class="container-fluid">` instead of Odoo's default `<div class="container">` elements.

[Bootstrap relevant docs](http://getbootstrap.com/css/#overview-container).

### CODE REVIEW

File: website_container_fluid/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+This module was written to extend the functionality of the website editor to
```
Comment (pedrobaeza) [inline website_container_fluid/README.rst:7]: Module title missing

File: website_container_fluid/static/description/icon.svg [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (pedrobaeza) [inline website_container_fluid/static/description/icon.svg]: As this icon is the sample one without modifications, you don't need to provide it.

**Comment by pedrobaeza** [comment]
Can you provide a sample block with this kind of container inserted in one demo page?

**Comment by yajo** [comment]
All done.

File: website_sale_collapse_categories/static/src/css/website_sale.css [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/exec/base.rb:52:in `parse'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/exec/base.rb:19:in `parse!'
+/usr/local/bundle/gems/sass-3.4.16/bin/sass:13:in `<top (required)>'
+/usr/local/bundle/bin/sass:23:in `load'
+/usr/local/bundle/bin/sass:23:in `<main>'
+*/
+body:before {
+  white-space: pre;
+  font-family: monospace;
+  content: "Error: Inconsistent indentation: 1 space was used for indentation, but the rest of the document was indented using 4 spaces.\A         on line 9 of /app/website/website_sale_collapse_categories/static/src/css/website_sale.sass\A \A 4:             display: inline-block\A 5:             width: 80%\A 6:             padding-left: 3px\A 7: \A 8:         i.fa\A 9:  cursor: pointer"; }
```
Comment (dreispt) [inline website_sale_collapse_categories/static/src/css/website_sale.css]: ???

Comment (yajo) [inline website_sale_collapse_categories/static/src/css/website_sale.css]: Oops, Sass auto-copiler complaining. I should not have committed that. Fixed now.

**Comment by pedrobaeza** [comment]
Tried on runbot and everything seems right :+1: 

Travis error is unrelated.

**Comment by LeartS** [comment]
Testing on runbot I get this graphic issue:
ui-problem
Which I do not get on runbot of the 8.0 branch of this repo.

Apart from that, :+1:

**Comment by yajo** [comment]
I fixed that glitch, but now I need help with the checks. Seems like some UI tests from upstream module `website_sale` fail, but I do not understand those tests nor why do they fail. The same did fail before last commit, so I guess it's not related to what @LeartS said.

**Comment by pedrobaeza** [comment]
Don't worry about Travis. It's broken for now for this repo. If @LeartS gives their second approval, then I merge.

**Comment by dgzurita** [comment]
:+1: very useful

**Comment by LeartS** [comment]
:+1:

**Comment by yajo** [comment]
Thanks!

**Pull request merged**